jwst/resample/resample.py

```diff
@@ -89,6 +88,9 @@ def __init__(self, input_models, output=None, single=False, blendheaders=True,
 
         if pscale is not None:
             pscale /= 3600.0
+            log.info(f'Output pixel scale: {pscale} deg.')
```

Comment:
Because the user is supplying the parameter value in units of arcsecs, perhaps it would be better to report it in the log while it's still in those units, rather than after it gets converted to degrees (just so the user has direct confirmation that the value they provided is the one being used). That could be done by simply moving this up 1 line before `pscale` gets converted.

Reply:
done